tests/e2e/map/map_controls.spec.js

```diff
@@ -284,40 +284,39 @@ test.describe('Map: Control interactions', () => {
     })
   })
 
-  test.describe.skip('When changing the map layers', () => {
-    test.describe('When changing the base layer to Blue Marble', () => {
+  test.describe('When changing the map layers', () => {
+    test.describe('When changing the base layer to World Imagery', () => {
       test('updates the URL with the new map parameter and updates the src of tile images', async ({ page }) => {
         await interceptUnauthenticatedCollections({
           page,
           body: commonBody,
           headers: commonHeaders
         })
 
-        await page.goto('/')
+        await page.goto('/search?base=trueColor')
+
+        // Set up the response promise BEFORE interacting with the UI
+        const responsePromise = page.waitForResponse((response) => response.url().includes('wi.maptiles.arcgis.com/arcgis/rest/services/World_Imagery/MapServer'))
 
-        // Change the base layer
-        await page.getByRole('button', {
-          name: 'Layers',
-          exact: true
-        }).hover({ force: true })
+        // Look for the layer switcher button by its class and aria-label
+        await page.locator('button[aria-label="Layer Options"]').hover({ force: true })
 
-        await page.getByRole('radio', { name: 'Corrected Reflectance (True Color)' }).click()
-        await page.getByRole('radio', { name: 'Blue Marble' }).click()
+        // Wait for the panel to become visible
+        await page.waitForSelector('.edsc-map-layer-switcher__panel--visible')
+
+        // Click the Land/Water Map radio button
+        await page.locator('input#layer-worldImagery').click()
```

Comment:
Can you use better selectors for these radio buttons and checkboxes? We want our tests to behave in a way a user would, and a user does not look for inputs by the ID we give them. Does `getByLabel('World Imagery)` work?